include/wx/msw/setup0.h

```diff
@@ -463,6 +463,15 @@
 // Recommended setting: 1 (but may be safely disabled if you don't use it)
 #define wxUSE_SECRETSTORE   1
 
+// Allow the use of the OS built-in spellchecker on wxTextCtrl (multiline
+// only). Currently wxGTK, wxMSW and wxOSX implemented.
```

Comment:
Where is the implementation for wxMSW?

Reply:
As mentioned on Trac, I adapted the GTK version only to implement the new API as VZ wasn't happy with the original API I proposed. 

Once the API is agreed, I will port the changes to wxMSW and OSX.

Reply:
..so until then you should probably enable it only for the platform where it is implemented. When another implementation is added enable that platform too.

Reply:
Noted, restriction to implemented platforms added in the new pull request #2473.